pex/commands/command.py

```diff
@@ -0,0 +1,266 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Moved from pex/tools/command.py below save for PEXCommand: https://github.com/pantsbuild/pex/pull/1459/files#diff-53a0c89163a0852f29460cfda14daf24bba9612dc102f11888e4b5b1f2d47fb5L149